python-backend/app/api/mines/status/resources/status.py

```diff
@@ -1,28 +1,15 @@
 from flask_restplus import Resource
 from werkzeug.exceptions import BadRequest, NotFound
 
-from ..models.mine_status import MineStatus
 from app.api.mines.status.models.mine_status_xref import MineStatusXref
 from app.extensions import api
 from ....utils.access_decorators import requires_role_view_all
```

Comment:
Should these Mixins and Decorators be absolute as well?

Reply:
I'm going to make another PR targeting the destruction of the error mixin.